index.bs

```diff
@@ -6670,7 +6674,9 @@ dictionary MLLstmSupportLimits {
   MLTensorLimits peepholeWeight;
   MLTensorLimits initialHiddenState;
   MLTensorLimits initialCellState;
-  MLTensorLimits outputs;
+  MLTensorLimits output0;
+  MLTensorLimits output1;
+  MLTensorLimits output2;
```

Comment:
Yeah, this is an improvement, since the outputs share the same data type but can vary for rank.